## 1 Задание.
Случайная непрерывная величина A имеет равномерное распределение на промежутке (200, 800].
Найдите ее среднее значение и дисперсию.


In [35]:
a = 200
b = 800

In [36]:
Mx = (a+b)/2
Mx

500.0

In [37]:
Dx = ((b-a)**2) / 12
Dx

30000.0

#### Среднее = 500
#### Дисперсия = 30000


## 2 Задание.
О случайной непрерывной равномерно распределенной величине B известно, что ее дисперсия равна 0.2.
Можно ли найти правую границу величины B и ее среднее значение зная, что левая граница равна 0.5?
Если да, найдите ее.


In [38]:
Dx = 0.2
a = 0.5

Выразим b из формулы дисперсии.

In [39]:
b = (12*Dx)**0.5 + a
b

2.049193338482967

In [40]:
Mx = (a+b) / 2
Mx

1.2745966692414834

#### Правая граница = 2.049193338482967
#### Среднее = 1.2745966692414834

## 3 Задание. 
Непрерывная случайная величина X распределена нормально и задана плотностью распределения
f(x) = (1 / (4 * sqrt(2*pi))) * (exp(-(x+2)**2) / 32).
Найдите:
а). M(X)
б). D(X)
в). std(X) (среднее квадратичное отклонение)


#### Среднее = 2
#### Стандартное отклонение = 4
#### Дисперсия = 16

## 4 Задание.

4. Рост взрослого населения города X имеет нормальное распределение.
Причем, средний рост равен 174 см, а среднее квадратичное отклонение равно 8 см.
Какова вероятность того, что случайным образом выбранный взрослый человек имеет рост:

а). больше 182 см

б). больше 190 см

в). от 166 см до 190 см

г). от 166 см до 182 см

д). от 158 см до 190 см

е). не выше 150 см или не ниже 190 см

ё). не выше 150 см или не ниже 198 см

ж). ниже 166 см.


In [41]:
std = 8
Mx = 174

In [42]:
Z1 = (182 - Mx) / std
Z1

1.0

In [43]:
P = 1 - 0.8413 #т.к считаем "Больше", то отнимаем от 1
P

0.15869999999999995

При Z = 1, P = 0.1586

In [44]:
Z2 = (190 - Mx) / std
Z2

2.0

In [45]:
P2 = 1 - 0.9772 #т.к считаем "Больше", то отнимаем от 1
P2

0.022800000000000042

In [46]:
Z31 = (166 - Mx) / std
Z31 #P = 0.1587

-1.0

In [47]:
Z32 = (190 - Mx) / std
Z32 # P = 0.9772

2.0

In [48]:
P3 =  0.9772 - 0.1587 # Получаем промежуток вероятности между 2 вероятностями
P3

0.8185

In [49]:
P4 = 0.8413 - 0.1587 # Получаем промежуток вероятности между 2 вероятностями
P4

0.6826000000000001

In [50]:
Z51 = (158 - Mx) / std
Z51 # P = 0.0228

-2.0

In [51]:
P5 = 0.9772 - 0.0228 # Получаем промежуток вероятности между 2 вероятностями
P5

0.9543999999999999

In [52]:
Z61 = (150 - Mx) / std
Z61 # P = 0.00135

-3.0

In [53]:
P6 = 0.00135 + (1-0.9772)
P6

0.024150000000000043

In [54]:
Z72 = (198 - Mx) / std
Z72 # P = 0.99865

3.0

In [55]:
P7 = 0.00135 + (1-0.99865)
P7

0.0026999999999999624

In [56]:
Z8 = (166 - Mx) / std
Z8

-1.0

P8 = 0.1587

#### Ответы:
- а) Р = 0.1587
- б) Р = 0.0228
- в) Р = 0.8185
- г) Р = 0.6826
- д) Р = 0.9544
- е) Р = 0.02415
- ё) Р = 0.0027
- ж) Р = 0.1587

## 5 Задание
На сколько сигм (средних квадратичных отклонений) отклоняется рост человека, равный 190 см, от
математического ожидания роста в популяции, в которой M(X) = 178 см и D(X) = 25 кв.см?


In [57]:
x = 190
std = 5
Mx = 178

In [58]:
Z = (x-Mx) / std
Z

2.4

### Ответ: на 2.4 сигмы